From 70d9f923dccdecab1df8abcd5a6a4715557f487a Mon Sep 17 00:00:00 2001
From: Mathieu Duponchelle <mathieu@centricular.com>
Date: Fri, 13 Jul 2018 17:21:25 +0200
Subject: [PATCH 47/72] avcfg: use g_object_get_property, not g_object_getv

I simply confused those two, we didn't need g_object_getv
here anyway.
---
 ext/libav/gstavcfg.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ext/libav/gstavcfg.c b/ext/libav/gstavcfg.c
index 5152024..ce9edd6 100644
--- a/ext/libav/gstavcfg.c
+++ b/ext/libav/gstavcfg.c
@@ -573,6 +573,7 @@ gst_ffmpeg_cfg_fill_context (GObject * object, AVCodecContext * context)
       continue;
 
     g_object_getv (object, 1, &pspec->name, &value);
+    g_object_get_property (object, pspec->name, &value);
     set_option_value (context, pspec, &value, opt);
   }
   g_free (pspecs);
-- 
2.17.0

